scripts/create_cita_config.py

```diff
@@ -479,33 +421,31 @@ def parse_arguments():
         help='The data is encrypted and transmitted on the network')
 
     # enable get version
-    pcreate.add_argument(
-        '--enable_version',
-        action='store_true',
-        help='Jsonrpc will return cita version')
+    pcreate.add_argument('--enable_version',
+                         action='store_true',
+                         help='Jsonrpc will return cita version')
 
     # switch to stdout
-    pcreate.add_argument(
-        '--stdout',
-        action='store_true',
-        help='Logs will output to stdout')
+    pcreate.add_argument('--stdout',
+                         action='store_true',
+                         help='Logs will output to stdout')
 
     #
     # Subcommand: append
     #
 
-    pappend = subparsers.add_parser(
-        SUBCMD_APPEND, help='append a node into a existed chain')
+    pappend = subparsers.add_parser(SUBCMD_APPEND,
+                                    help='append a node into a existed chain')
 
-    pappend.add_argument(
-        '--chain_name', required=True, help='Name of the existed chain.')
+    pappend.add_argument('--chain_name',
+                         required=True,
+                         help='Name of the existed chain.')
 
-    pappend.add_argument(
-        '--node',
-        required=True,
-        type=NetworkAddressList.from_str_get_one,
-        metavar='IP:PORT',
-        help='Node network addresses for new nodes.')
+    pappend.add_argument('--node',
+                         required=True,
+                         type=NetworkAddressList.from_str_get_one,
+                         metavar='IP:PORT',
+                         help='Node network addresses for new nodes.')
 
     pappend.add_argument(
```

Comment:
Why is this line not formatted?

Reply:
I don't know why, maybe `yapf` makes something wrong. I will use `pylint`.

Reply:
You can check it on next pr. use yapf or not